src/main/java/seedu/taassist/model/moduleclass/StudentModuleData.java

```diff
@@ -18,6 +18,7 @@
  */
 public class StudentModuleData implements Identity<StudentModuleData> {
 
+    private static final double DUMMY_GRADE = 0.0;
```

Comment:
This doesn't seem to be utilized anywhere in the code, perhaps consider removing?

Reply:
This was used in another revision which got reverted. I will remove it!